src/index.js

```diff
@@ -114,6 +114,17 @@ inquirer.prompt(questionnaire).then(async (answers) => {
         }
       );
     }
+    if (isDark && frontEndChoice === 'angular') {
+      fsExtra.copy(
+        `${currentPath}/themeTemplates/angular-themes`,
+        `${frontEnd.path}/src/angular-themes`,
+        (err) => {
+          if (err) {
+            console.log("Error Found:", err);
```

Comment:
Let's update error messages to be more specific & while keeping them short as well